flask_unchained/bundles/oauth/views/oauth_controller.py

```diff
@@ -0,0 +1,40 @@
+from flask import abort, request, session
+from flask_unchained import (
+    Controller, injectable, lazy_gettext as _, route, url_for)
+from http import HTTPStatus
+
+from ..extensions import OAuth
+
+
+class OAuthController(Controller):
+    oauth: OAuth = injectable
```

Comment:
Just noticed you added the `OAuthService` class, but it's not getting used by the controller. Adding that would look about like this:

```python
# ...
from .services import OAuthService

class OAuthController(Controller):
    oauth: OAuth = injectable
    oauth_service: OAuthService = injectable

    def authorized(self, remote_app):
        # ...
        session['oauth_token'] = resp['access_token']
        self.oauth_service.on_authorized(provider)
        # ...
```